index.html

```diff
@@ -72,8 +70,106 @@ <h3 class="ability_header">Skills</h3>
+                    <h3>Multi-Post Stories</h3>
+                    <p class="work_info">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="languages">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button type="button" class="btn">See Project</button>
+                </div>
+            </div>
+
+            <div class="work_four">
+                <img src="placeholder.png" alt="image placeholder">
+                <div class="supporting_text">
+                    <h3>Multi-Post Stories</h3>
+                    <p class="work_info">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="languages">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button type="button" class="btn">See Project</button>
+                </div>
+            </div>
+            <div class="contact_us">
+                <h2>CONTACT US</h2>
+                <p class="work_info">
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding. 
+                    I'd love to help with it 
+                </p>
+                <form name="contact_form">
+                    <input type="text" placeholder="Name">
+                </form>
+                <form name="email">
+                        <input type="text" placeholder="Email">
+                    </form>
+                    <form name="text">
+                        <input class="customer_info" type="text" placeholder="What can we help you with">
+                    </form>
+                </div>
+                <div class="btn_links">
+                    <button type="button" class="btn">See Project</button>
+                    <ul class="social_media_links">
+                        <li class="left_border"><i class="fa-brands fa-github"></i></li>
+                        <li><i class="fa-brands fa-linkedin"></i></li>
+                        <li><i class="fa-brands fa-twitter"></i></li>
+                        <li><i class="fa-brands fa-angellist"></i></li>
+                    </ul>
+                </div>
+            </section>
```

Comment:
- Kindly add the `email` as it is visible to the provided Figma design. 😉 

  - ### Figma design
  ![lololol](https://user-images.githubusercontent.com/87197412/185690685-d00073b9-e5d0-49cd-8c37-ad921f1ede6e.PNG)


    - ### Your design
  ![rals](https://user-images.githubusercontent.com/87197412/185690604-d65d765c-4515-4da6-94fb-09edf96467ff.PNG)

Reply:
- kindly buy trying to fix this email requires by the previous review you add another bag

> Kindly add the email as it is visible to the provided Figma design.


- kindly make sure you fix this, your icons are not showing well, they combine with the title of  your card 
![image](https://user-images.githubusercontent.com/95297251/185714034-a14d2b20-ec79-4de5-9ad2-1ec91aeda69f.png)